bin/npm-force-upgrade.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+var path = require('path')
+var join = path.join
+var resolve = path.resolve
+var read = require('fs').readFileSync
+var exec = require('child_process').execSync
+var args = process.argv.slice(2)
+var argv = require('minimist')(args, {
```

Comment:
Do we actually need `minimist` here? Since this this is the only dep and it's only used for meta information, I'd prefer to not have it.

Reply:
that's true, i just added it for the `--package` arg which is probably unnecessary in retrospect. it's not actually a big deal though since this is like a global abstraction and not a dev dependency. will still remove